src/main/pg/gps.c

```diff
@@ -32,7 +32,11 @@
 PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 4);
 
 PG_RESET_TEMPLATE(gpsConfig_t, gpsConfig,
+#if defined(USE_VIRTUAL_GPS)
```

Comment:
```suggestion
#if defined(SITL) && defined(USE_VIRTUAL_GPS)
```

@haslinghuis : GPS_VIRTUAL is IMO better default for SITL